tests/test_publish_success.py

```diff
@@ -72,10 +73,11 @@ def test_publish_success(self):
 
         # WHEN `dpm publish` is invoked
         result = self.invoke(cli, ['publish', '--publisher', 'testpub'])
-
         # THEN 'publish ok' should be printed to stdout
         self.assertRegexpMatches(result.output, 'publish ok')
-        # AND 5 requests should be sent
+        #Checking README
+        self.assertIn('Uploading README', result.output)
```

Comment:
We should also have tests for cases 
- When there is no README at all
- When README does not have extension at all (should be uploaded)
- When README has wrong name Eg: READMEE or README. (not sure about last one)
- When readme is lowercase